packages/core-toggle/core-toggle.js

```diff
@@ -84,7 +84,16 @@ export default class CoreToggle extends HTMLElement {
   }
 
   // aria-haspopup triggers forms mode in JAWS, therefore store as custom attr
-  get popup () { return this.getAttribute('popup') === 'true' || this.getAttribute('popup') || this.hasAttribute('popup') }
+  get popup () {
+    const hasDeprecatedPopup = this.getAttribute('popup') === 'true' || this.getAttribute('popup') || this.hasAttribute('popup')
+    if (hasDeprecatedPopup) {
```

Comment:
Could we rather have just `this.hasAttribute('popup')` as the `hasDeprecatedPopup` variable, and later return 
```
const key = hasDeprecatedPopup ? 'popup' : 'data-popup'
return this.getAttribute(key) === 'true' || this.getAttribute(key) || this.hasAttribute(key)
```
as this clarifies the purpose? :)

Reply:
Sure, that also feels less bloated. Stay tuned